### DISCUSSION TITLE
How to display a TreeListView with columns?

### DISCUSSION THREAD

**Discussion by aohashi084** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3474] [Q&A] [answered]
How do I display a TreeListView with columns like the following?

https://www.codeproject.com/Articles/24973/TreeListView-2

**Comment by Keboo** [comment, marked as answer]
At present there is not a built in control with this capability. The new `TreeListView` has the potential of this being added but at present it doesn't exist.

**Reply by aohashi084** [reply]
Thank you.
